test/Microsoft.OpenApi.Tests/Models/References/OpenApiTagReferenceTest.cs

```diff
@@ -45,6 +45,7 @@ public class OpenApiTagReferenceTest
           description: The user was not found.
       tags:
         - $ref: '#/tags/user'
+        - users.user
```

Comment:
I believe The case above with $ref is invalid? 
Unless that's like a negative test to make sure the ref syntax does not get caught by some logic when we don't want it to?

Reply:
I think I understand what you mean here. The name of the `tag` should not be a ref to the tag at the root of the document, but should simply match the name as tags at the operation level are defined as a list of strings. 
Yes?